cli/src/pcluster/models/cluster.py

```diff
@@ -285,7 +285,8 @@ def compute_fleet_status(self) -> ComputeFleetStatus:
     def login_nodes_status(self):
         """Status of the login nodes pool."""
         login_nodes_status = LoginNodesStatus(self.stack_name)
-        login_nodes_status.retrieve_data()
+        if self.config.login_nodes:
+            login_nodes_status.retrieve_data()
         return login_nodes_status
```

Comment:
Do we need to return an empty LoginNodesStatus object if we don't use login nodes?

Reply:
Yes, basically the LoginNodesStatus exposes the information about the fact that there are no load balancers available.